packages/host/app/components/operator-mode.gts

```diff
@@ -47,8 +47,8 @@ import OperatorModeStateService from '@cardstack/host/services/operator-mode-sta
 
 interface Signature {
   Args: {
-    firstCardInStack: Card;
     onClose: () => void;
+    updateOperatorModeStateQueryParams: () => void;
```

Comment:
I'm wondering if there's a way to update query params from `OperatorModeStateService` so that we wouldn't have to pass this action from the controller. Maybe calling `transitionTo` with the updated query param would work the same.

Reply:
Paired with @ef4 on this - would make more sense to change the query param from the state service after manipulation happens. We can get the controller with the `lookup` method.